build-tools/create-packs/Microsoft.NET.Workload.Android.proj

```diff
@@ -9,15 +9,14 @@ workload manifest pack containing information about the various Microsoft.Androi
 <Project Sdk="Microsoft.Build.NoTargets">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Just an FYI, it seems like it doesn't matter what `$(TargetFramework)` is set to for a `Microsoft.Build.NoTargets` project?

https://github.com/microsoft/MSBuildSdks/tree/master/src/NoTargets

`$(TargetFramework)` just has to be valid and not blank... Maybe there is a way we could this value from a `Directory.Build.targets` at some point? Then we wouldn't need it in these files.

Reply:
I think this is only used when packing, but we ignore it mostly since we set explicit paths for everything in the packs... We can try to centralize it in the future and reduce the warnings coming from our pack invocations at the same time.